lib/localisation/locales.ts

```diff
@@ -1,2 +1,9 @@
 // ! Remember to modify the middleware.ts file as you add or remove locales here
-export const SUPPORTED_LOCALES = ['en', 'es'] as const;
+export const SUPPORTED_LOCALES = ['en', 'es', 'ar'] as const;
+
+export type Locales = (typeof SUPPORTED_LOCALES)[number];
```

Comment:
I think it would be better to name this as `Locale`, I know i named it as `Locales` before. Now, realised it while reviewing :smile:

Reply:
resolved [bfe2dc0](https://github.com/complexdatacollective/Studio/pull/35/commits/bfe2dc0dd4babf2a51598673261c574e03afb0c0)